pkgs/build-support/fetchgithub/default.nix

```diff
@@ -17,6 +18,9 @@ let
     else builtins.unsafeGetAttrPos "rev" args
   );
   baseUrl = "https://${githubBase}/${owner}/${repo}";
+  newPassthru = passthru // {
+    inherit rev owner repo;
+  };
```

Comment:
Hmmm, I am unsure about the ordering here....

Reply:
Now the order is `owner`, `repo`, and `rev`.

Reply:
I didn't mean the inherit, that doesn't matter, I mean if we should overwrite a potential passed in rev/owner/repo or not.

Reply:
Oops! I see. Let's give developers the power to shadow those attributes if they want to.

Reply:
Addressed.

Reply:
Would you be available to take a look at it again?